src/components/tests/Balance.spec.js

```diff
@@ -21,3 +21,10 @@ test("Balance renders Link account component when Linked Account is false", () =
   console.log(wrapper.props().className);
   expect(wrapper.props().className).toBe("makeStyles-NoBalance-1");
 });
+
+
+test('Balace renders correctly',()=>{
```

Comment:
Can we fix the typo on the test description here?

Reply:
Let's add a comment about what this test will eventually be, in the format of // TODO: Test future balance component